packages/shopping/cypress/integration/cart.spec.js

```diff
@@ -0,0 +1,52 @@
+context('Cart', () => {
+  before(() => cy.visit('http://localhost:3000/shoppy.html'));
+
+  it('should add and checkout all items in Cart', () => {
+    cy.get('#products')
+      .find('.card')
+      .first()
+      .get('.action-buttons')
+      .contains('Details')
+      .click();
+
+    addToCart();
+
+    cy.get('#shoppingCartLink')
+      .click()
+      .then(() =>
+        // we need to wait for the pop-up to show with the items to checkout
+        // if there are more items in Cart the wait time might need to be increased
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy
+          .get('.list-group')
+          .find('li')
+          .wait(500)
```

Comment:
We really do not want `.wait()` if we can avoid it.  Please try something like:
````
        cy
          .get('.list-group')
          .find('li')
          .get('input[name="cartItem"]')
          .should('be.visible')
          .check(),
````
Good article here: https://cypress.io/blog/2019/01/22/when-can-the-test-click/

Reply:
Okay cool will update. Thanks